extensions/blocks/contact-form/components/jetpack-contact-form.js

```diff
@@ -201,13 +209,21 @@ class JetpackContactForm extends Component {
 		const formClassnames = classnames( className, 'jetpack-contact-form', {
 			'has-intro': ! hasFormSettingsSet,
 		} );
+		const { limited } = getJetpackExtensionAvailability(
+			this.props.name.replace( 'jetpack/', '' )
+		);
 
 		return (
 			<Fragment>
 				<InspectorControls>
 					<PanelBody title={ __( 'Email feedback settings', 'jetpack' ) }>
 						{ this.renderToAndSubjectFields() }
 					</PanelBody>
+					{ ! limited && (
```

Comment:
We'll revert these changes before merging this, this is just to make it possible to test and grok what the intent of the change is.